ManPac/Assets/Scripts/Player/Death/GhostStateHandler.cs

```diff
@@ -0,0 +1,26 @@
+using UnityEngine;
+using UnityEngine.Events;
+
+
+[RequireComponent(typeof(DeathHandler))]
+public class GhostStateHandler : MonoBehaviour
+{
+    private DeathHandler thisDeathHandler;
+    private void Start()
```

Comment:
can you add a whitespace here
```suggestion
    private DeathHandler thisDeathHandler;
    
    private void Start()
```